renku/service/serializers/datasets.py

```diff
@@ -122,7 +122,7 @@ class DatasetListRequest(Schema):
 
 
 class DatasetDetails(Schema):
-    """Serialize dataset to response object."""
+    """Serialize a dataset to response object."""
```

Comment:
```suggestion
    """Serialize a dataset to a response object."""
```

Reply:
My linter says both forms are correct. :thinking:

Reply:
run the grammar check in word to break the tie :)

Reply:
I don't have to. I trust you and your years of experience of being grammar police! I applied the requested changes!